packages/dds/pact-map/src/interfaces.ts

```diff
@@ -15,6 +15,25 @@ export interface IPactMapEvents extends ISharedObjectEvents {
 	(event: "pending" | "accepted", listener: (key: string) => void);
 }
 
+/**
+ * Details of the accepted pact.
+ */
+export interface IAcceptedPact<T> {
+	/**
+	 * The accepted value of the given type or undefined (typically in case of delete).
+	 */
+	value: T | undefined;
+
+	/**
+	 * The sequence number when the value was accepted, which will normally coincide with one of two possibilities:
+	 * - The sequence number of the "accept" op from the final client we expected signoff from
+	 * - The sequence number of the ClientLeave of the final client we expected signoff from
+	 *
+	 * For values set in detached state, it will be 0.
```

Comment:
Suggest leaving off these details for the public-facing interface - in the future we may add explicit accept/delay/reject API surface which would change the specific triggers.

```suggestion
	 * The sequence number when the value was accepted.
	 *
	 * For values set in detached state, it will be 0.
```